packages/docs/content/components/dropdown.mdx

```diff
@@ -89,7 +89,7 @@ The following CSS variables can be overridden to customize Dropdown components:
   - I use the arrow keys to select an option and I hear the selected option is changed.
   - I select an option with the Enter or Space key and hear the selected option.
 
-<!-- [We need to implement this] I use the escape key and I hear it collapse and focus moves to the select -->
+{/* [We need to implement this] I use the escape key and I hear it collapse and focus moves to the select */}
```

Comment:
Original comment syntax broke migrations, so I had to change these files to complete them.